http-add-on/README.md.gotmpl

```diff
@@ -120,16 +120,6 @@ their default values.
   {{- end }}
 {{- end }}
 
-### Webhooks
-
-| Parameter | Type | Default | Description |
-|-----------|------|---------|-------------|
-{{- range .Values }}
-  {{- if hasPrefix "webhooks" .Key }}
-| `{{ .Key }}` | {{ .Type }} | {{ if .Default }}{{ .Default }}{{ else }}{{ .AutoDefault }}{{ end }} | {{ if .Description }}{{ .Description }}{{ else }}{{ .AutoDescription }}{{ end }} |
-  {{- end }}
-{{- end }}
-
 Specify each parameter using the `--set key=value[,key=value]` argument to
```

Comment:
Why do you remove this?

This is the README config/template for the other chart

Reply:
It's because that chart doesn't have any webhook. Only `keda` char has

Reply:
Yes, because it should not be there :)

There are no webhooks there so every helm-doc run it adds empty things